etcd/src/test/java/org/jclouds/etcd/features/KeysApiLiveTest.java

```diff
@@ -94,6 +94,40 @@ public void testDeleteNonExistentKey() {
       assertNull(deletedKey);
    }
 
+   @Test
+   public void testWaitKey() {
+      String localKey = randomString();
+      String localValue = randomString();
+      Key createdKey = api().createKey(localKey, localValue, 3);
+      assertNotNull(createdKey);
+      assertNotNull(createdKey.node().expiration());
+      assertTrue(createdKey.node().ttl() == 3);
+
+      createdKey = api().waitKey(localKey);
```

Comment:
By default methods aren't executed with specific timeouts; just the HTTP connection timeouts controlled by the timeout properties in the jclouds `Constants` class.